musicbot/__init__.py

```diff
@@ -117,7 +118,9 @@ def _add_logger_level(levelname, level, *, func_name=None):
 log = logging.getLogger(__name__)
 log.setLevel(logging.EVERYTHING)
 
-fhandler = logging.FileHandler(filename="logs/musicbot.log", encoding="utf-8", mode="a")
+# added code for handing logs with dates in the name
+musicbotlog = datetime.now().strftime('logs/musicbot_%Y-%m-%d_%H-%M.log')
```

Comment:
We can probably move the date and time to constants.py as STARTTIME, rather than checking the current date and time in every file. Just to ensure any and all uses of the log file use the same file, and don't try to make a new one because they were a second late on a slower device